config.toml

```diff
@@ -16,11 +16,13 @@ pluralizeListTitles = false
   keywords = ["adopters", "eclipse adopters", "eclipse project adopters"]
   gcse = "011805775785170369411:p3ec0igo0qq"
   js = "assets/js/scripts.min.js"
+  hugo_js = ""
```

Comment:
Please fix the JS path rather than zero it out

Reply:
weird, is it still showing empty on your side? I just updated

Reply:
I sent this as you were changing it which is why it shows the old version I believe!